Monika After Story/game/zz_selector.rpy

```diff
@@ -2778,14 +2794,29 @@ label monika_clothes_select:
         )
         sel_map = {}
 
+        def_clothes = [mas_selspr.CLOTH_SEL_MAP["def"]]
```

Comment:
outside of the selspr store, we shouldn't access the maps directly. There is a function `get_sel` that retrieves selector for a sprite object.
```suggestion
        def_clothes = [mas_selspr.get_sel(mas_clothes_def)]
```

Reply:
This line can go because of my reply to the next comment